pandas/tests/io/parser/test_c_parser_only.py

```diff
@@ -680,3 +680,10 @@ def test_float_precision_options(c_parser_only):
 
     with pytest.raises(ValueError, match=msg):
         parser.read_csv(StringIO(s), float_precision="junk")
+
+
+def test_invalid_file_inputs(c_parser_only):
+    # GH#45957
+    parser = c_parser_only
```

Comment:
maybe also add pyarrow?

Reply:
thx, I tend to forget the pyarrow engine...